packages/editor/src/components/media-placeholder/index.js

```diff
@@ -53,26 +52,8 @@ class MediaPlaceholder extends Component {
 
 	onSubmitSrc( event ) {
 		event.preventDefault();
-		if ( this.state.src ) {
-			if ( this.props.onSelectUrl ) {
-				// TODO: In removing deprecation, ensure to simplify rendering
-				// to avoid checking for `onSelectUrl`. It also allows this
-				// function to be simplified to avoid truthiness test on
-				// `onSelectURL`, since it's required for the form invoking
-				// this function to be rendered at all.
-				deprecated( 'MediaPlaceholder `onSelectUrl` prop', {
-					alternative: '`onSelectURL` prop',
-					plugin: 'Gutenberg',
-					version: 'v3.5',
-					hint: 'The prop has been renamed.',
-				} );
-
-				this.props.onSelectUrl( this.state.src );
-			}
-
-			if ( this.props.onSelectURL ) {
-				this.props.onSelectURL( this.state.src );
-			}
+		if ( this.state.src && this.props.onSelectURL ) {
```

Comment:
The comment above is a bit unclear to me, but seems to imply we shouldn't (need to) check for truthiness of `onSelectUrl`. It says it's required for this function to run, so do we still want to check for it?

Reply:
I double checked this one, this prop is option, it is good as is.